src/vcpkg/spdx.cpp

```diff
@@ -244,11 +244,12 @@ static void find_all_distfile(StringView text, Json::Array& packages)
             it = text.end();
             continue;
         }
-        auto url = extract_arg_from_cmake_invocation_args(distfile, CMakeVariableUrls);
-        auto filename = extract_arg_from_cmake_invocation_args(distfile, CMakeVariableFilename);
+        auto url = fix_ref_version(extract_arg_from_cmake_invocation_args(distfile, CMakeVariableUrls), version_text);
+        auto filename =
+            fix_ref_version(extract_arg_from_cmake_invocation_args(distfile, CMakeVariableFilename), version_text);
         auto sha = extract_arg_from_cmake_invocation_args(distfile, CMakeVariableSHA512);
         packages.push_back(make_resource(
-            fmt::format("SPDXRef-resource-{}", packages.size()), filename, url.to_string(), sha, filename));
+            fmt::format("SPDXRef-resource-{}", packages.size()), filename, std::move(url), sha, filename));
```

Comment:
As far as I know fmt never moves from one of the args so this isn't a meaningful move.

Reply:
The move is not inside format. url goes directly into `make_resource`